libs/platform/src/lib/table/components/table-row/table-row.component.html

```diff
@@ -30,14 +30,14 @@
         >
             <!-- hidden text for screenreader -->
             <span
-                [ngStyle]="{
-                    position: 'absolute !important',
-                    height: '1px',
-                    width: '1px',
-                    overflow: 'hidden',
-                    clip: 'rect(1px, 1px, 1px, 1px)',
-                    color: 'transparent'
-                }"
+                style="
```

Comment:
This will bring back this issue: https://github.com/SAP/fundamental-ngx/issues/10347